src/plotconfig.jl

```diff
@@ -183,6 +184,18 @@ function PlotConfig(T::Val{:designmat})
     return cfg
 end
 
+function PlotConfig(T::Val{:splines})
+    cfg = PlotConfig()
+    config_kwargs!(
+        cfg;
+        layout = (;),
+        axis = (; ylabel = "Density value"),
```

Comment:
I was wondering how we deal with subplots and PlotConfig; would it make sense to have two PlotConfig entries for each subplot one? 

In general, how would you go about to change e.g.  the ylabel of the "Spline value" subplot=